src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final User user = new User();
+    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();

```

Comment:
```suggestion
    private final RegistrationServiceImpl registrationService;
```
better initialize in method with annotation Before or BeforeClass (depends on Junit version)